src/theme.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
```

Comment:
```suggestion
```
this is unused in the file, so you wont need this

Reply:
Fixed.